modules/yahoosspBidAdapter.js

```diff
@@ -590,7 +590,7 @@ export const spec = {
         adId: deepAccess(bid, 'adId') ? bid.adId : bid.impid || bid.crid,
         adUnitCode: bidderRequest.adUnitCode,
         requestId: bid.impid,
-        bidderCode: spec.code,
+        bidderCode: bidderRequest.bidderCode,
```

Comment:
Wouldn't it be better to just remove this line ? It will be filled automatically by Prebid with the correct value (it avoids eventual incompatiblity with this method in the future)

Reply:
Hey @JulieLorin  - A unit test was added to ensure we don't regress here. If this line is dropped the unit test will fail. I _guess_ we could test for bidderCode to be undefined. It's a little odd, but works.